server.ts

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const fallback = require('express-history-api-fallback');
+
+const app = express();
+const PORT = 3000;
```

Comment:
При развертывании приложения на разных машинах в докере бывает полезно указать порт через переменные окружения
```suggestion
const PORT = process.env.PORT || 3000;
```